backend/app/resources/validate_utils.py

```diff
@@ -186,9 +184,7 @@ def validate_donation_info(donation_info, error_list):
                 error_list.append(
                     "The commitment_date supplied is not a datetime object."
                 )
-            if val < datetime.now():
```

Comment:
This error check was poorly thought out (when I wrote it hehe) since sometimes we often want to set the `commitment_date` to `utc.now()`, in which case by the time this is checked, this condition will evaluate to true and an error will be thrown. For this reason, it is removed.